_sass/layouts/_homepage.scss

```diff
@@ -11,6 +11,17 @@
     margin: 0 0 25px;
   }
 
+  .project-title {
+    color: #fff;
+    // font-size: 1.2em;
+    font-size: 46px;
+    font-weight: bold;
+  }
+
+  .project-subtitle {
+    color: #ffc400;
```

Comment:
Question, feel free to save this for another PR. Can we make the colors from the color palette variables that way we don't need to update them everywhere?